src/icons/svgs/visibility_24px_rounded.svg

```diff
@@ -0,0 +1,5 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
no more `viewbox`, `height`, or `width`. Those are now specified on the outer svg.

Reply:
ok